### PR TITLE
Reacts & Redux Capstone Project Week 5 - Covid 19 Worldometers

### PR DESCRIPTION
**Pull request by 187jjay187**
### The following features have been implemented:

- [Video Presentation Link](https://www.loom.com/share/32bf4f17b78f40b9adb0ec8b945f63f8)
- [Live Demo](https://worldometers.onrender.com) - if live demo link gives 502 error it is still being deployed
- Covid-19 API has been used - https://disease.sh/v3/covid-19/countries.
- All the design guidelines have been followed.
- When the page loads, the web app shows the list of categories/countries with parameters that can be filtered out through the search.
- The application navigates to the details page when the user clicks on a category/country item.
- On the details page, the web app retrieves data from the API to show detailed data in the category/country.
- When the user clicks on the back icon, the user navigates to the home page.
- Created unit tests for pure functions in the Redux code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,212 @@
...
+- search countries using the rearch field (search is key specific)
+- when a country is clicked it renders a list of details from the API
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Live Demo Link - ([Covid-19-worldometers]())
```
Comment (cynthiainga) [inline README.md]: - [x] Please kindly provide the correct link to live demo, the link that you provided doesn’t work 🙃 🙃

File: MIT.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Jay G.O.
```
Comment (cynthiainga) [inline MIT.md]: - [OPTIONAL] I think it's a good idea if you update your MIT file to look more professional 😎

File: README.md [hunk 3]
```diff
@@ -1 +1,212 @@
...
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <img src="https://github.com/microverseinc/readme-template/blob/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
```
Comment (cynthiainga) [inline README.md]: - [x] Please kindly delete the Microverse logo or if you have your logo you can add it here and delete the Microverse one 🙃 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @187jjay187, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [x] No linters errors ✔️ 
- [x] Descriptive PR ✔️ 
- [x] Correct Gitflow  ✔️ 
- [x] Good design ✔️ 

## Required Changes ♻️

- [x] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (187jjay187) [reply README.md]: Good day @cynthiainga😃 I've made the above, required changes that you've asked me to make to the README.md file, if possible could I please ask you for another code review 🙏 

kind regards
Jay

Comment (cynthiainga) [reply README.md]: Hi @187jjay187, if everything is okay just ask for the second review in your dashboard and your project will be approved 😉

**Review by uwadonat — APPROVED** [review]
## Hi @187jjay187,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Professional readme file ✔️
- Amazing design ✔️
- No linter errors ✔️
- Well organized and clean project ✔️
approval


# Status: Approved ✔️✔️✔️


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**